tutorial.ipynb

```diff
@@ -172,12 +156,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "df['full_content'] = (df.description + df.title)\n",
+    "# df['full_content'] = (df.description + df.title)\n",
+    "df['full_content'] = (df.title)\n",
```

Comment:
This is quite confusing. We can just use `df.title` in the notebook and then switch to `df.full_content` later to show the incremental benefit.